src/falconpy/_payload/_incidents.py

```diff
@@ -41,6 +41,7 @@ def incident_action_parameters(passed_keywords: dict) -> list:
     """Create a properly formatted action_parameters branch for incident action payload operations.
 
     Available keywords
+    add_comment - Adds the associated value as a new cmoment on all the incidents in the ids list.
```

Comment:
Spelling, not sure if this is coming in from generated docs

Reply:
Doh! Nice catch, correcting now.